source/includes/kubernetes/_k8_statefulsets.md

```diff
@@ -0,0 +1,334 @@
+                "volumeMounts": [
+                  {
+                    "mountPath": "/etc/aerospike",
+                    "name": "config-volume"
+                  }
+                ]
+              }
+            ],
+            "dnsPolicy": "ClusterFirst",
+            "restartPolicy": "Always",
+            "schedulerName": "default-scheduler",
+            "securityContext": {},
+            "terminationGracePeriodSeconds": 30,
+            "volumes": [
+              {
+                "configMap": {
+                  "defaultMode": 420,
+                  "items": [
+                    {
+                      "key": "aerospike.conf",
+                      "path": "aerospike.conf"
+                    }
+                  ],
+                  "name": "test-aerospike"
+                },
+                "name": "config-volume"
+              }
+            ]
+          }
+        },
+        "updateStrategy": {
+          "rollingUpdate": {
+            "partition": 0
+          },
+          "type": "RollingUpdate"
+        }
+      },
+      "status": {
+        "collisionCount": 0,
+        "currentReplicas": 1,
+        "currentRevision": "test-aerospike-6db7776c7d",
+        "observedGeneration": 1,
+        "readyReplicas": 1,
+        "replicas": 1,
+        "updateRevision": "test-aerospike-6db7776c7d",
+        "updatedReplicas": 1
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/statefulsets</code>
+
+Retrieve a list of all stateful sets in a given [environment](#administration-environments).
+
+| Attributes                                 | &nbsp;                                                          |
+| ------------------------------------------ | --------------------------------------------------------------- |
+| `id` <br/>*string*                         | The id of the stateful set                                      |
+| `metadata` <br/>*object*                   | The metadata of the the stateful set                            |
+| `metadata.creationTimestamp` <br/>*string* | The date of creation of the stateful set as a string            |
+| `metadata.labels` <br/>*map*               | The labels associated to the stateful set                       |
+| `metadata.name` <br/>*string*              | The name of the stateful set                                    |
+| `metadata.namespace` <br/>*string*         | The namespace in which the stateful set is created              |
+| `metadata.uid` <br/>*object*               | The UUID of the stateful set                                    |
+| `spec`<br/>*object*                        | The specification used to create and run the stateful set       |
+| `status`<br/>*object*                      | The status information of the the stateful set                  |
```

Comment:
Additional `the` can be removed.

Reply:
e56d9f4fa4cc2bdb50c51822c11b30f5af7f81fd